src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -21,6 +21,10 @@ public static class SchemaVersionConstants
         public const int SupportsPartitionedResourceChangeDataVersion = (int)SchemaVersion.V20;
         public const int AddPrimaryKeyForResourceTable = (int)SchemaVersion.V25;
 
+        // Added TextHash column to StringSearchParameter and TokenStringCompositeSearchParam.
+        // SystemId, QuantityCodeId and RespurceTypeId nullable columns are marked as NOT NULL. Respective tables are back-filled with index 0 instead of null values
+        public const int AddPrimaryKeyPart2 = (int)SchemaVersion.V26;
```

Comment:
Can we update comment, and change name to something like: AddPrimaryKeyForSearchParameterTables?